snapcraft/cli/_command.py

```diff
@@ -32,6 +32,11 @@ def run_legacy_snapcraft(argv=sys.argv[1:]) -> None:
             "Install snapcraft from https://snapcraft.io/snapcraft and try again."
         )
 
+    # Set PYTHONPATH so python loads the correct packages.
+    os.environ["PYTHONPATH"] = os.path.join(
```

Comment:
keeping sitecustomize would make this unnecessary, right?

Reply:
indeed, gone.